### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by thetechnick**
Signed-off-by: Nico Schieder <nschieder@redhat.com>

### CODE REVIEW

**Comment by thetechnick** [comment]
@bentito This should make it easier to get started and try out PKO. :)
We'll improve on these docs when we are closer to stable (Somewhere around end of next month :crossed_fingers: )

File: README.md [hunk 1]
```diff
@@ -5,13 +5,78 @@
...
+- [Features](#features)
+- [Documentation](#documentation)
+- [Contributing](#contributing)
+- [Development](#development)
+
+---
+
+## Project Status
+
+Package Operator is currently in early development and not neither complete nor ready for production use.
```
Comment (yashvardhan-kukreja) [inline README.md:34]:
```suggestion
Package Operator is currently in early development and neither complete nor ready for production use.
```

**Comment by yashvardhan-kukreja** [comment]
just the above thing @thetechnick , rest all looks great!

File: README.md [hunk 2]
```diff
@@ -5,13 +5,78 @@
...
+- [Features](#features)
+- [Documentation](#documentation)
+- [Contributing](#contributing)
+- [Development](#development)
+
+---
+
+## Project Status
+
+Package Operator is currently in early development and neither complete nor ready for production use.
```
Comment (Ajpantuso) [inline README.md:34]:
```suggestion
Package Operator is currently in early development and is neither complete nor ready for production use.
```

File: README.md [hunk 3]
```diff
@@ -5,13 +5,78 @@
...
+APIs are not yet final and breaking changes might be introduced in every release.
```
Comment (Ajpantuso) [inline README.md:35]:
```suggestion
APIs have not been finalized and are subject to breaking changes with future releases.
```

File: README.md [hunk 4]
```diff
@@ -5,13 +5,78 @@
...
+	- Plug and Play
+- Cheap Failures and Easy Recovery
+	- Rollout History
+	- Rollback
+
+## Documentation
+
+Package Operator documentation is available on [package-operator.run](https://package-operator.run).
+
+The source of this website is our [website repository](https://github.com/package-operator/package-operator.github.io) which is hosted via Github Pages, [Hugo](https://gohugo.io/) and using the [Doks template](https://getdoks.org/).
```
Comment (Ajpantuso) [inline README.md:53]:
```suggestion
The source of this website is our [website repository](https://github.com/package-operator/package-operator.github.io) which is hosted via Github Pages, generated with [Hugo](https://gohugo.io/) and uses the [Doks template](https://getdoks.org/).
```

File: README.md [hunk 5]
```diff
@@ -5,13 +5,78 @@
...
+
+## Contributing
+
+Thank you for taking time to help to improve Package Operator!
+
+Feel free to open issues or propose pull requests, but please keep in mind that Package Operator is still in early development and response times for accessing contributions may vary.
```
Comment (Ajpantuso) [inline README.md:59]:
```suggestion
Feel free to open issues or propose pull requests, but please keep in mind that Package Operator is still in early development and response times for reviewing contributions may vary.
```

I thinks that was supposed to be `assessing`, but reviewing seems more familiar in this context.

File: README.md [hunk 6]
```diff
@@ -5,13 +5,78 @@
...
+
+We also are not yet ready to open project management and meetings to the general public.
```
Comment (Ajpantuso) [inline README.md:61]:
```suggestion
Project management and meetings are not open to the general public at this time.
```

File: README.md [hunk 7]
```diff
@@ -5,13 +5,78 @@
...
+## Development
+
+The Package Operator project keeps almost all dependencies confined to the project checkout directory and manages dependencies within the project itself.
+
+To get started hacking, you just need Go 1.18+ and podman or docker installed.
+
+### Commands and local development
+
+> **Dev Note**
+> Assure you have `export CONTAINER_RUNTIME=docker` or similar for `podman`, or you will get cryptic errors from mage that may lead you to think there is a problem with the kind cluster.
```
Comment (Ajpantuso) [inline README.md:72]:
```suggestion
> Before running _mage_ targets run `export CONTAINER_RUNTIME=docker`, `export CONTAINER_RUNTIME=podman` if using `podman`, or you will get cryptic errors from mage that may lead you to think there is a problem with the kind cluster.
```

File: magefile.go [hunk 8]
```diff
@@ -106,32 +108,37 @@ func (Test) Integration(ctx context.Context) error {
...
 
+// Build all PKO binaries for the architecture of this machine.
 func (Build) Binaries() {
 	mg.Deps(
 		mg.F(Builder.Cmd, "package-operator-manager", "linux", "amd64"),
 		mg.F(Builder.Cmd, "mage", "", ""),
 	)
 }
 
+// Builds only the given container image, building binaries as prerequisite as required.
```
Comment (Ajpantuso) [inline magefile.go:119]:
```suggestion
// Builds the given container image, building binaries as prerequisite as required.
```

**Review by Ajpantuso — COMMENTED** [review]
Some recommendations for clarity, but all are optional.

**Pull request merged**